source/Calamari.ConsolidateCalamariPackages.Tests/Calamari.ConsolidateCalamariPackages.Tests.csproj

```diff
@@ -17,12 +17,12 @@
     </ItemGroup>
     <ItemGroup>
         <!-- These references are imported so that the Integration test has something to work with -->
-        <PackageReference Include="Calamari" Version="12.0.2" GeneratePathProperty="true" />
-        <PackageReference Include="Calamari.Cloud" Version="12.0.2" GeneratePathProperty="true" />
-        <PackageReference Include="Sashimi.Terraform" Version="0.1.7-enh-packcalamari0002" GeneratePathProperty="true" />
```

Comment:
I'm a bit concerned about why this version is being used and when exactly the version was built. There are a lot of differences in the files and file hashes in the *.approved.txt files as a result of the Sashimi.Terraform change